### ISSUE TITLE
After last update (1.1.4) arm64e stuck on deep black screen and solve only after hard reboot

### ISSUE THREAD

**Issue by porroso** [opa334/libSandy#5]
After remove of arm64e slice from sandyd got this issue, with version 1.1.3 was all good but now no way to roll back using cydia . testing on iPhone 11

**Comment by opa334** [comment]
Seems unrealistic that the update could cause that, you can manually download 1.1.3 from [here](http://apt.thebigboss.org/repofiles/cydia/debs2.0/libsandy_1.1.3.deb) and install it, but I doubt doing so will fix the issue. Let me know if it does though.

**Comment by porroso** [comment]
Yes opa334, strange to me too, btw thank you for 1.1.3 version link, after 3 days with that version I solved my black screen problem. I will retry to update again to see if issue happen again. The only change I made was the libsandy update, no others new packages installed. Issue solved for me with 1.1.3, thanks again.

**Comment by opa334** [comment]
How is that a solution???

**Comment by porroso** [comment]
Using 1.1.6 and no problem at all now , was not an issue, was something related to my phone, so all version are ok, sorry for miss understanding
